src/Data/Profunctor.purs

```diff
@@ -33,5 +34,11 @@ rmap b2c = dimap id b2c
 arr :: forall a b p. (Category p, Profunctor p) => (a -> b) -> p a b
 arr f = rmap f id
 
+wrapIso :: forall t a p. (Newtype t a, Profunctor p) => p t t -> p a a
+wrapIso = dimap wrap unwrap
+
+unwrapIso :: forall t a p. (Newtype t a, Profunctor p) => p a a -> p t t
```

Comment:
Ah good point, probably. I was going to ask though - do we need both versions here? I know you can turn a profunctor iso lens around, but you need to bring lenses in for that, right?